staging/src/k8s.io/pod-security-admission/admission/api/register.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name use in this package
+const GroupName = "apiserver.config.k8s.io"
```

Comment:
Interesting that this group gets split across multiple repos.  Not strictly a problem.  This one is specific to a kube-apiserver versus generic apiservers though.

Reply:
I was planning for the same config file to be able to be consumed by the standalone webhook

Reply:
> I was planning for the same config file to be able to be consumed by the standalone webhook

The API is still kube specific, right?  As in, the configuration here only makes sense when applied to a level of the kube-apiserver pods. Versus the existing apiserver.config.k8s.io which makes sense when applied to any generic apiserver.

Not a point I feel very strongly about, but it did surprise me a bit.

Reply:
huh... I could have sworn that we had other kubeapiserver admission config types using this group, but... I couldn't find any

Reply:
hmm... I don't like any of the prior art for group names for admission plugins here:

Plugin-specific groups (doesn't end in `.config.k8s.io`, and a group per plugin seems excessive):
* eventratelimit.admission.k8s.io
* podtolerationrestriction.admission.k8s.io
* resourcequota.admission.k8s.io

Current group (as you point out, a pod-oriented admission plugin is specific to kube-apiserver):
* apiserver.config.k8s.io

Reply:
went with `pod-security.config.admission.k8s.io`

* `pod-security`: matches format of the label keys for the feature
* `admission`: since this is an `admission` component that may or may not be inside an API server
* `config.k8s.io`: matches convention for configuration file types vs REST API types